frontend/package.json

```
@@ -9,8 +9,12 @@
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
+    "http": "http-server dist/corona-game -p 4200 --proxy http://localhost:4200?",
     "postinstall": "npm run version",
     "prebuild": "ts-node --project bin/tsconfig.json bin/build-lang-files.ts -d src/assets/i18n/ && npm run version",
+    "cypress": "node_modules/.bin/cypress open",
```

would not this be enough?
```suggestion
    "cypress": "cypress open",
```